jf_agent/main.py

```diff
@@ -132,6 +132,20 @@ def main():
 
     else:
         jellyfish_endpoint_info = obtain_jellyfish_endpoint_info(config, creds)
+        if jellyfish_endpoint_info.jf_options:
```

Comment:
NIT:

If you default `jf_options` to an empty dict in `obtain_jellyfish_endpoint_info` you might be able to simplify this block and not do as much null checking. This also might be a good fail safe incase something upstream changes tho

Reply:
swapped here: https://github.com/Jellyfish-AI/jf_agent/pull/248/commits/2bfdbd50c418a9336b588881aebbe9e79c13ce57#diff-b9f32c609e4bd39398d8a39e8f65ae5978f4b9db7a7c023eaa470ee438043373R136